sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakeFileSystemAsyncClient.java

```diff
@@ -466,7 +466,31 @@ public HttpPipeline getHttpPipeline() {
      */
     public Mono<DataLakeFileAsyncClient> createFile(String fileName) {
         try {
-            return createFileWithResponse(fileName, null, null, null, null, null).flatMap(FluxUtil::toMono);
+            return createFile(fileName, false);
+        } catch (RuntimeException ex) {
```

Comment:
Do we need to duplicate catching the RuntimeException in every overload?

Reply:
Just caught a few in this class - I just pushed a commit removing them.

I left some where we flatMap(FluxUtil::toMono) cause it seems to make sense there? What do you think?